keras_nlp/src/models/csp_darknet/csp_darknet_backbone.py

```diff
@@ -39,7 +39,7 @@ class CSPDarkNetBackbone(Backbone):
             `"basic_block"` for basic conv block.
             Defaults to "basic_block".
         image_shape: tuple. The input shape without the batch size.
-            Defaults to `(None, None, 3)`.
```

Comment:
actually I think this is what we want, as long as the backbone does actually support different sized inputs. so maybe update the default below?